### ISSUE TITLE
Add credentials to person

### ISSUE THREAD

**Issue by opoudjis** [relaton/relaton-bib#83]
https://github.com/metanorma/metanorma-nist/issues/277 depends on Relaton recognising `credential` in the following grammar:

````
person = 
  element person {
    fullname?, credential*, affiliation*, person-identifier*, contact*
}

credential = element credential { text }
````

As far as I can see, it isn't recognised.

**Comment by opoudjis** [comment]
In addition, we have the convention in hashes that we supply the `content` node for a string:

`a.is_a?(String) ? FormattedString.new(content: a) : a`

Please extend that to affiliation_hash_to_bib:

`name = LocalizedString.new(a[:name][:content], a[:name][:language], a[:name][:script]) if a[:name]`

should be

````
if a[:name]
  name = a[:name].is_a?(String) ? FormattedString.new(content: a[:name]) : a
end
````

so that I can specify in hash:

````
  person:
    name:
      completename: Fred Flintstone
    affiliation:
    - name: Vice President, Medical Devices Quality & Compliance -- Strategic programmes
````

instead of

````
  person:
    name:
      completename: Fred Flintstone
    affiliation:
    - name: 
        content: Vice President, Medical Devices Quality & Compliance -- Strategic programmes
````

**Comment by andrew2net** [comment]
> In addition, we have the convention in hashes that we supply the `content` node for a string:

@opoudjis FYI we have that convention for parsing only. When rendering we using `content` because the `person/affiliation/name` node is `LocalizedString` and can have attributes. The `relaton-py` project can't parse variant without `content`.

**Comment by opoudjis** [comment]
Sure, but the context here is recognising YAML, so it is parsing.

**Comment by andrew2net** [comment]
Fixed in relaton-bib 1.17.3
